### ISSUE TITLE
IJRR data and analyses

### ISSUE THREAD

**Issue by surberj** [surberj/Vision#21]
On google drive I can find only the first indoor dataset of the three published sets. Todo: Ping Stefan Leutenegger or Simon Lynen again for the other 2 sets. It would be very useful to have the outdoor sets as well.

Results of ROVIO on the first set (red: ground truth, blue: ROVIO):
![screenshot from 2015-11-18 21 00 12](https://cloud.githubusercontent.com/assets/9253897/11252716/ef4d909e-8e37-11e5-80a4-f539b09cf655.png)

Published OKVIS results on the set:
![screenshot from 2015-11-18 21 01 30](https://cloud.githubusercontent.com/assets/9253897/11252719/f184375a-8e37-11e5-8dc3-0ac27b8123d2.png)

remarks:
- the orientation error is erroreous because based on euler angles (but it is in the right order of magnitude)
- on such a long dataset the drift problem becomes very visible!
- It surprises me that the msckf did such a good job. I think Rovio is set up well (parameters, ...) but still shows worse performance.

**Comment by surberj** [comment]
![screenshot from 2015-11-18 20 18 19](https://cloud.githubusercontent.com/assets/9253897/11272783/b5a75c7e-8ecf-11e5-9bf6-261f6f7cdc2a.png)

**Comment by surberj** [comment]
After cleanup of error calculation, I got now clean results for two calibrations - firstly the one of the IJRR dataset, secondly the one lucas collected with the newer VI-Sensor:

ROVIO with IJRR calibration (red: ground truth, blue: rovio):
![ijrr](https://cloud.githubusercontent.com/assets/9253897/11411251/adcc6ebe-93ce-11e5-8630-ef19a034fda2.png)

ROVIO with lucas calibration:
![screenshot from 2015-11-21 20 43 26](https://cloud.githubusercontent.com/assets/9253897/11323090/3008c612-9108-11e5-9505-5a1a2f88e895.png)

OKVIS - published:
![screenshot from 2015-11-18 21 01 30](https://cloud.githubusercontent.com/assets/9253897/11323144/afe4f840-910a-11e5-8a3a-e083c69a2c93.png)

Remarks:
- I changed calculation of orientation error from euler angles to dcm (Got rid off outliers but the errors are nearly the same).
- I underestimated the influence of working with a camera-calibration done on another VI-Sensor. For this dataset the error is 2 times higher with lucas calibration.
- The Drift issue is still obvious. On another dataset (not running in circles) the orientation error of 30 - 40 degrees would leed to huge position errors.
- With ROVIO and the IJRR calibration something seems to have happend around 700 meters travelled distance: The orientation error (and therefore also the position error) increase there overproportionally.
